doc/_static/mpl.css

```diff
@@ -1118,3 +1118,28 @@ div.viewcode-block:target {
 .sidebar-announcement p {
     margin: 0.4em 0.4em 0.6em 0.4em;
 }
+
+/* new main nav */
+nav.main-nav{
+    background-color: #002b47;
+    font-family: Calibri, 'Gill Sans', 'Gill Sans MT', 'Trebuchet MS', sans-serif;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    font-size: 11px;
```

Comment:
Elsewhere font sizes are specified as ems.  That makes it easier to re-size the whole page if we want.  Any reason to hard-code the sizes here?

Reply:
Modern web browsers will zoom in on pixel-sized text correctly these days (especially since we're all looking at the web with devices with different screen resolutions). Ems are more useful these days when building layouts that scale with parent containers. 

I do appreciate that you're thinking about website accessibility though.